custom_md_loader/classes/AppConstants.cls

```diff
@@ -7,11 +13,13 @@ public class AppConstants {
     public static final String DESC_ATTRIBUTE = 'Description';
     public static final String MDT_SUFFIX = '__mdt';
     
+    public static final String CS_NAME_ATTRIBURE = 'Name';
+    
     public static final String SELECT_STRING = 'Select type';
     
     
     //error messages
```

Comment:
One of the things I wanted to do longer-term was put these errors in actual label files, but we can do that later too.

Reply:
Makes sense, I will take a stab using labels.